web_client/templates/panels/annotationSelector.pug

```diff
@@ -11,7 +11,7 @@ block content
     .btn-group.btn-group-sm(role='group')
       button.btn.btn-default.h-hide-all-annotations(type='button', title='Hide all annotations')
         | #[span.icon-eye-off]
-    span(title='Annotation opacity')
+    span.h-annotation-opacity-container(title=`Annotation opacity ${opacity * 100}%`)
```

Comment:
Can you use `toFixed()`?  I see values like `Annotation opacity 7.000000000000001%`

Reply:
That's odd, the slider uses `step="0.01"`.  I wasn't seeing that behavior, but I'll add it to be sure.